cdc/processor/pipeline/system/system.go

```diff
@@ -15,19 +15,28 @@ package system
 
 import (
 	"context"
+	"fmt"
+	"sync"
 
+	"github.com/pingcap/ticdc/cdc/model"
 	"github.com/pingcap/ticdc/pkg/actor"
 )
 
 // System manages table pipeline global resource.
 type System struct {
 	tableActorSystem *actor.System
 	tableActorRouter *actor.Router
+	idLock           sync.Mutex
```

Comment:
Could add some comments for these id related fields? Like watch this ids stand for.

Reply:
done